fe_plugins/auditloader/src/main/java/org/apache/doris/plugin/audit/DorisStreamLoader.java

```diff
@@ -123,7 +127,13 @@ public LoadResponse loadBatch(StringBuilder sb) {
         HttpURLConnection beConn = null;
         try {
             // build request and send to fe
-            feConn = getConnection(loadUrlStr, label);
+            if (slowLog) {
+                label = "slow" + label;
```

Comment:
```suggestion
                label = "slow_" + label;
```

Reply:
An underscore is already included in “label".